components/locale/src/fallback/algorithms.rs

```diff
@@ -239,13 +235,6 @@ mod tests {
             expected_language_chain: &["en-US-u-sd-usca", "en-US", "en-u-sd-usca", "en"],
             expected_region_chain: &["en-US-u-sd-usca", "en-US", "und-US-u-sd-usca", "und-US"],
         },
-        TestCase {
-            // TODO(#4413): -u-rg is not yet supported; when it is, this test should be updated
```

Comment:
Nit: keep a reference to #4413

Reply:
The design doc doesn't mention a `-u-rg` requirement